tekton/integration_pipeline.go

```diff
@@ -171,3 +176,34 @@ func (r *IntegrationPipelineRun) WithApplicationAndComponent(application *applic
 
 	return r
 }
+
+// WithEnvironmentAndDeploymentTarget adds a param containing the DeploymentTarget connection details and Environemtn name
```

Comment:
typo: `Environemtn` -> `Environment`

Reply:
Thanks, I realize I ofter have typo for this `environment`

Reply:
Yeah, I have the same issue, thanks for correcting it